std/http/file_server.ts

```diff
@@ -6,7 +6,7 @@
 // TODO Add tests like these:
 // https://github.com/indexzero/http-server/blob/master/test/http-server-test.js
 
-const { ErrorKind, cwd, args, stat, readDir, open } = Deno;
+const { ErrorKind, DenoError, cwd, args, stat, readDir, open } = Deno;
```

Comment:
You can revert this change it seems.

Reply:
I used [here](https://github.com/denoland/deno/pull/3467/files#diff-b6647ec3bad62bae632704dd0c54e7ceR145)